apps/frontend/src/renderer/components/task-detail/TaskMergedChanges.tsx

```
@@ -0,0 +1,383 @@
+          </ScrollArea>
+        </div>
+      )}
+
+      {/* Files section with diff viewer */}
+      {files.length > 0 && (
+        <div className="space-y-2">
+          <button
+            onClick={() => setShowFiles(!showFiles)}
+            className="text-sm font-medium flex items-center gap-2 hover:text-foreground text-foreground/80 transition-colors"
+          >
+            {showFiles ? (
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)